pkg/virt-api/validating-webhook/validating-webhook.go

```diff
@@ -47,6 +47,10 @@ const (
 )
 
 var validInterfaceModels = []string{"e1000", "e1000e", "ne2k_pci", "pcnet", "rtl8139", "virtio"}
+var macAddressRegex = regexp.MustCompile(
+	// there are other MAC schemes (e.g. with dashes instead of colons),
```

Comment:
it's actually arguable if we want just colons-form. Chime in with comments if you think we should support different forms.

Reply:
@fabiand what do you think about supporting other MAC address formats known to golang net package? (Like DE-AD-00-00-BE-AF.) Do we want to support them? If so, do we store them as-is or convert to a canonical form? If not, do we want to make the rules even more strict, f.e. forcing lower-case letters?

Reply:
I think we need to support any mac address format just call the net.ParseMAC and check for error.